plugin/pxr/doc/README.md

```diff
@@ -32,6 +32,8 @@ The plugin creates two commands: `usdImport` and `usdExport`, and will also regi
 | ---------- | --------- | ---- | ------- | ----------- |
 | `-api` | `-apiSchema` | string (multi)| none | Imports the given API schemas' attributes as Maya custom attributes. This only recognizes API schemas that have been applied to prims on the stage. The attributes will properly round-trip if you re-export back to USD. |
 | `-ar` | `-assemblyRep` | string | `Collapsed` | If the import results in the creation of assembly nodes, this value specifies the assembly representation that will be activated after creation. If empty, no representation will be activated. Valid values are: `Collapsed`, `Expanded`, `Full`, `Import`, `(empty)`. `Import`: No USD reference assembly nodes will be created, and the geometry will be imported directly. `(empty)`: The assembly is created but is left unloaded after creation. See "Importing as Assemblies" below for more detail on assembly node creation. |
+|`-chr` | `-chaser` | string(multi) | none | Specify the import chasers to execute as part of the export. See "Import Chasers" below.
```

Comment:
We will no longer update this document. Please add this documentation to https://github.com/Autodesk/maya-usd/blob/dev/lib/mayaUsd/commands/Readme.md

Reply:
This is addressed in 3c48ae3.